### PR TITLE
chips and dialog added

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Screenshot:
chips-and-dialog

## Deploy Preview Link: 
https://deploy-preview-12--comp-ui.netlify.app/

### CODE REVIEW

File: components/Dialog/dialog.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.dialog {
+    width: 30%;
+    height: max-content;
+    background-color: #b4d6d2;
```
Comment (yashpatel1797) [inline components/Dialog/dialog.css]: use variable for colors.

File: components/chips/chips.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    margin: 0 0.7em 0 -1.5em;
+    height: 3em;
+    width: 3em;
+    border-radius: 50%;
+}
+.closebtn{
+    font-size:1.5em;
+    padding-left: 0.5em;
+    cursor: pointer;
+    background-color:#d1e7dd;
```
Comment (pnchinmay) [inline components/chips/chips.css]: change it to color variable

**Review by pnchinmay — CHANGES_REQUESTED** [review]

**Comment by pnchinmay** [comment]
Looks good

**Review by yashpatel1797 — APPROVED** [review]
Looks good to me.

**Review by pnchinmay — APPROVED** [review]

**Pull request merged**